engine/bindings/python/interface.h

```diff
@@ -0,0 +1,34 @@
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+#include <string>
+#include <memory>
+#include "db/db_server.hpp"
+static PyObject *EpsillaError;
+
+static PyObject *spam_system(PyObject *self, PyObject *args);
+static PyObject *load_db(PyObject *self, PyObject *args);
+static PyObject *use_db(PyObject *self, PyObject *args);
+static PyObject *create_table(PyObject *self, PyObject *args);
+static PyObject *insert(PyObject *self, PyObject *args);
+static PyObject *query(PyObject *self, PyObject *args);
+
+static std::string db_name;
+static vectordb::engine::DBServer *db;
+
+static PyMethodDef EpsillaMethods[] = {
+    {"system", spam_system, METH_VARARGS, "Execute a shell command."},
+    {"load_db", load_db, METH_VARARGS, "Load the database"},
+    {"use_db", use_db, METH_VARARGS, "Use the database"},
```

Comment:
Why not all functions above exposed here?

Reply:
this is draft PR.

Reply:
PR is competed and everything is added